lib/include/spvgentwo/Instruction.h

```diff
@@ -498,6 +498,8 @@ namespace spvgentwo
 		Instruction* opSatConvertUToS(Instruction* _pSignedInt) { return scalarVecOp(spv::Op::OpSatConvertUToS, _pSignedInt, nullptr, "Operand of OpSatConvertUToS is not a scalar or vector of unsigned integer type"); }
 
 		// Instruction* OpConvertUToPtr(); TODO
+        Instruction* opConvertUToPtr(Instruction* _pUInt, const Type& _type);
```

Comment:
Please pass the result type as Instruction* first, in the same order as the operands are expected by the spec: Instruction* opConvertUToPtr(Instruction* _pResultType, Instruction* _pUInt);

Generally, non of the other instructions take a const Type& parameter because i expect the user  to reuse type instructions on the user side. It should always be possible for a user  to just look at the SPIR-V spec and infer the order and type of the instruction operands. So opXXX functions take either Instruction* or enum/literal values. the ResultType is omitted when SpvGentTwo can infer the type for all inputs.